rpcs3/util/atomic.hpp

```diff
@@ -222,6 +222,17 @@ namespace atomic_wait
 			return *this;
 		}
 
+		template <uint Index, op Flags = op::eq, typename T2, typename = std::void_t<decltype(std::declval<T2>().template wait<op::eq>(any_value))>>
```

Comment:
If unused, remove please. Waiting on value should be explicit, we only use default value in constructor because of no choice (or no good ideas how to enforce it).